.github/workflows/release-release-artifacts.yml

```
@@ -300,177 +300,29 @@ jobs:
+      fdroid: ${{ needs.resolve.outputs.fdroid == 'true' }}
+      linux: ${{ needs.resolve.outputs.linux == 'true' }}
+      windows: ${{ needs.resolve.outputs.windows == 'true' }}
+      macos: ${{ needs.resolve.outputs.macos == 'true' }}
+      ios: ${{ needs.resolve.outputs.ios == 'true' }}
+      android_apk: ${{ needs.resolve-artifacts.outputs.android_apk }}
+      vpnd_deb_x86_64: ${{ needs.resolve-artifacts.outputs.vpnd_deb_x86_64 }}
+      vpnd_deb_aarch64: ${{ needs.resolve-artifacts.outputs.vpnd_deb_aarch64 }}
+      vpnc_deb_x86_64: ${{ needs.resolve-artifacts.outputs.vpnc_deb_x86_64 }}
+      vpnc_deb_aarch64: ${{ needs.resolve-artifacts.outputs.vpnc_deb_aarch64 }}
+      linux_app_deb_x64: ${{ needs.resolve-artifacts.outputs.linux_app_deb_x64 }}
+      linux_app_deb_aarch64: ${{ needs.resolve-artifacts.outputs.linux_app_deb_arm64 }}
```

actionlint (1.7.12)

[error] input "linux_app_deb_aarch64" is not defined in "./.github/workflows/prepare-release-notes.yml" reusable workflow. defined inputs are "android_apk", "fdroid", "github_release_type", "ios", "linux", "linux_app_appimage_arm64", "linux_app_appimage_x64", "linux_app_deb_arm64", "linux_app_deb_x64", "macos", "macos_installer_name", "version", "vpnc_deb_aarch64", "vpnc_deb_x86_64", "vpnc_macos_installer_name", "vpnd_deb_aarch64", "vpnd_deb_x86_64", "windows", "windows_app_installer_arm64", "windows_app_installer_x64"

(workflow-call)